lib/migration.js

```diff
@@ -66,10 +66,11 @@ function mixinMigration(MsSQL) {
     });
   };
 
-  MsSQL.prototype.isActual = function(cb) {
+  MsSQL.prototype.isActual = function(models, cb) {
```

Comment:
This is a breaking change to the api.

@kimkha Please consider changing the fix to this format the same as [mysql](https://github.com/strongloop/loopback-connector-mysql/blob/master/lib/migration.js#L124)